apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -232,6 +233,8 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
     @ViewChild('itemsContainer', { read: ViewContainerRef, static: false }) container: ViewContainerRef;
     /** Template reference for each entry */
     @ViewChild('entry', { read: TemplateRef, static: false }) template: TemplateRef<any>;
+    @ViewChild('AmountFieldComponent', { static: true }) public amountField: ElementRef;
```

Comment:
Please check if it is used as there is no reference in the TS file. If it is used then add JS doc

Reply:
removed